docs/installation.rst

```diff
@@ -67,7 +67,8 @@ You'll need to install Docker and Docker-compose on your development machine:
 *  `Docker-compose`_
 
 .. important:: 
-  We recommended Windows users to run `docker inside WSL <https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers>`_ (preferbly WSL2) to avoid any compatibility issues.
+  
+  We recommended Windows users to :doc:`docker-setup` (preferbly WSL2) to avoid any compatibility issues.
```

Comment:
"We recommended Windows users to running docker inside WSL" sounds a bit weird.
I see you set up auto-labeling but "to run docker inside WSL" would make more sense.
We can maybe change the header of docker-setup.rst ?